### DISCUSSION TITLE
problems flashing sonoff POWR320D

### DISCUSSION THREAD

**Discussion by bryanb334** [arendst/Tasmota#19962] [Support request]
I have (I thought) successfully flashed the device.  Windows 10, Web flasher, Tasmota(english), ESP32. It connects and installs, finishes up, and says complete.  When apply mains, I get nothing.  No LEDs or web interface. If I power it with just the usb serial, I can connect to the serial output and I can see that it is spontaneously rebooting.  Here is the output:

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:184
load:0x40078000,len:12732
ho 0 tail 12 room 4
load:0x40080400,len:2908
entry 0x400805c4

00:00:00.003-215/51 HDW: ESP32-D0WD-V3 v3.0 
00:00:00.036-214/50 UFS: FlashFS mounted with 308 kB free
00:00:00.054 CFG: Loaded from File, Count 2
00:00:00.064 QPC: Count 1
00:00:00.486 BRY: Berry initialized, RAM used 5164 bytes
00:00:00.504 TFS: File '.drvset003' not found
00:00:00.512 Project tasmota - Tasmota Version 13.2.0.2(tasmota32)-2_0_14(2023-11-07T19:32:13)
00:00:00.818 WIF: WifiManager active for 3 minutes
ets Jul 29 2019 12:21:46

Does anyone have some suggestions on what I have done wrong?
Thanks!
Bryan

**Comment by sfromis** [comment]
The output you see on serial is perfectly normal, however many devices cannot get on Wifi only powered by the USB/serial adapter, and may crash when trying. With mains power it should be broadcasting an AP for you to connect to, and configure Wifi credentials first. Until you also have configured the device template, do not expect any LED activity.

**Comment by bryanb334** [comment]
I did try with mains and nothing ...

**Comment by bryanb334** [comment]
Was I using the correct firmware? Tasmota(english), ESP32?
